Firmware/ultralcd.cpp

```diff
@@ -6920,29 +6940,63 @@ bool lcd_selftest()
 
 	_progress = lcd_selftest_screen(testScreen::extruderFan, _progress, 3, true, 2000);
 #if (defined(FANCHECK) && defined(TACH_0))
-	_result = lcd_selftest_fan_dialog(0);
+	switch (lcd_selftest_fan_auto(0)){		// check extruder Fan
```

Comment:
Better than state "check extruder Fan" in comment would be to state this in code. (Have named value stating extruder Fan.)